CHANGES.md

```diff
@@ -28,6 +28,7 @@
 - Build and run tests for all selected packages when checking distribution tarball
   (#266, @NathanReb)
 - Improve trimming of the changelog to preserve the indentation of the list of changes. (#268, @gpetiot)
+- The data read from the url file must be trimmed before filling url.src field (#270, @gpetiot)
```

Comment:
Can you improve the changelog entry please, this won't speak to users!

Reply:
done